src/pages/account/resetPassword.tsx

```diff
@@ -1,19 +1,61 @@
 import styled from '@emotion/styled';
-import type { NextPage } from 'next/types';
+import type { GetServerSideProps, NextPage } from 'next/types';
+import * as api from 'api';
 import { ResetPasswordForm } from 'components/account';
 import { Seo } from 'components/common';
+import { PAGE_PATH } from 'constants/common';
+import { getQueryParam } from 'utils';
 
-const ResetPassword: NextPage = () => {
+interface PageProps {
+  email: string;
+  token: string;
+}
+
+const ResetPassword: NextPage<PageProps> = ({
+  email,
+  token,
+}: {
+  email: string;
+  token: string;
+}) => {
   return (
     <>
       <Seo title={'비밀번호 재성정 | a daily diary'} />
       <ContentWrapper>
-        <ResetPasswordForm />
+        <ResetPasswordForm email={email} token={token} />
       </ContentWrapper>
     </>
   );
 };
 
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const { query } = context;
+  const email = getQueryParam(query.email);
+  const token = getQueryParam(query.token);
+
+  const REDIRECT_LOGIN_PAGE_PROPS = {
+    redirect: {
+      destination: PAGE_PATH.account.login,
+      permanent: false,
+    },
+  };
+
+  if (email.length === 0 || token.length === 0) {
+    return REDIRECT_LOGIN_PAGE_PROPS;
+  } else {
+    try {
+      await api.tempTokenValidation({
+        email: email[0],
+        tempToken: token[0],
+      });
+
+      return { props: { email: email[0], token: token[0] } };
+    } catch (error) {
+      return REDIRECT_LOGIN_PAGE_PROPS;
+    }
+  }
```

Comment:
p3;
if 문에서 return 문이 있어서 else 문 제거해도 좋을 것 같습니다.

```suggestion
  if (email.length === 0 || token.length === 0) {
    return REDIRECT_LOGIN_PAGE_PROPS;
  }
  
  try {
    await api.tempTokenValidation({
      email: email[0],
      tempToken: token[0],
    });

    return { props: { email: email[0], token: token[0] } };
  } catch (error) {
    return REDIRECT_LOGIN_PAGE_PROPS;
  }
```